curated_transformers/models/architectures.py

```diff
@@ -148,7 +147,9 @@ def build_albert_transformer_model_v1(
             grad_scaler_config=grad_scaler_config,
         )
 
-    piece_encoder = build_sentencepiece_encoder()
+    piece_encoder = (
+        build_sentencepiece_encoder_v1() if piece_encoder is None else piece_encoder
+    )
```

Comment:
Can't we have the default `v1` in the config as default value instead of `None`? Then we wouldn't have this hidden default here.

Reply:
(same comment for other places in this PR)

Reply:
Fixed.